recipes/hpp-gui/meta.yaml

```
@@ -0,0 +1,83 @@
+{% set name = "hpp-gui" %}
+{% set version = "4.10.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/humanoid-path-planner/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 677d3bd69e98c0a158a2525d36baf2c0e022968bf6e3a92f3e690d5eb0689393
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+    - doxygen
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libselinux-devel') }}      # [linux]
+    - {{ cdt('libxdamage-devel') }}      # [linux]
+    - {{ cdt('libx11-devel') }}          # [linux]
+    - {{ cdt('libxfixes') }}             # [linux]
+    - {{ cdt('libxxf86vm') }}            # [linux]
+    - {{ cdt('libxext') }}               # [linux]
+  host:
+    - qt
+    - boost
+    - boost-cpp
+    - tinyxml
+    - hpp-fcl
+    - hpp-util
+    - hpp-statistics
+    - hpp-pinocchio
+    - hpp-constraints
+    - hpp-core
+    - hpp-template-corba
+    - hpp-corbaserver
+    - hpp-manipulation-corba
+    - openscenegraph
+    - openscenegraph-osgqt
+    - gepetto-viewer
+    - gepetto-viewer-corba
+    - omniorb
+    - omniorbpy
+    - python
+    - xorg-libx11
+  run:
+    - boost
+    - boost-cpp
+    - tinyxml
+    - omniorb
+    - omniorbpy
+    - python
+
+test:
+  requires:
+    - pkg-config
+  commands:
+    - pkg-config --modversion {{ name }}
+    - test -f $PREFIX/lib/gepetto-gui-plugins/hppwidgetsplugin.so     # [linux]
+    - test -f $PREFIX/lib/gepetto-gui-plugins/hppwidgetsplugin.dylib  # [osx]
+
+about:
+  home: https://github.com/humanoid-path-planner/{{ name }}
+  license: LGPL-3.0-or-later
+  license_file: COPYING
+  summary: '{{ name }} subpackage of Humanoid Path Planner suite'
+  description: |
+    HPP (Humanoid Path Planner) is a collection of software packages implementing path planning functionalities for a humanoid robot. The core functionalities are well suited to solve classical path planning problems. It can be extended to path planning for other types of robots like digital actors for instance.
```

This should be wrapped:
```suggestion
    HPP (Humanoid Path Planner) is a collection of software packages implementing
    path planning functionalities for a humanoid robot. The core functionalities are well
    suited to solve classical path planning problems. It can be extended to path planning
    for other types of robots like digital actors for instance.
```

---

Will do !